### PR TITLE
Feature/plugins packages

### PR DESCRIPTION
**Pull request by cecenz**
This PR is adding extra functionality for out of the box Django-Wagtail.

Added:
- Babel
- ES Linting
- Prettier
Along with updates to the Webpack file

### CODE REVIEW

File: django-wagtail/webpack.config.js [hunk 1]
```diff
@@ -1,61 +1,78 @@
...
+                    },
+                    'svgo-loader',
+                ],
+            },
+            {
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                use: [
+                    {
+                        loader: 'babel-loader', // creates style nodes from JS strings
```
Comment (ryami333) [inline django-wagtail/webpack.config.js]: Erroneous comment 😄

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": ["airbnb", "prettier", "prettier/react"],
+    "plugins": ["prettier"],
+    "rules": {
+        "prettier/prettier": "error"
+    }
+}
```
Comment (katie-day) [inline .eslintrc.json]: @ryami333 @cecenz @holloway @samanthaksanders this is up for discussion, but we have this eslint config - [eslint-plugin-springload](https://github.com/springload/eslint-plugin-springload) and we talk about the plugin in our questionnaire/docs but are not using it here. 

We need to either use the plugin/config here, and/or change the plugin to use these rules, or remove the plugin altogether and just have some docs somewhere. What do you guys think?

Comment (ryami333) [reply .eslintrc.json]: I agree, we should probably use `eslint-plugin-springload` instead, though it does need a bit of a spruce-up.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (ryami333) [inline .prettierrc]: Could you please use the brand spanking new [`prettier-config-springload`](https://www.npmjs.com/package/prettier-config-springload)?

**Pull request merged**